File: 剑指offer/easy/20210913/replaceSpace.cpp
class Solution {
public:
    string replaceSpace(string s) {
        string str;
        for(int i=0;i<s.size();i++){
            if(s[i]==' '){
                str+="%20";
            }else{
                str+=s[i];
            }
        }
        return str;
    }
};

File: Review/20211003/replaceSpace.go
package _0211003

func replaceSpace(s string) string {
	res:=""
	for i:=0;i<len(s);i++{
		if s[i]==' '{
			res+="%20"
		}else{
			res+=string(s[i])
		}
	}
	return res
}
